web/impact/impact/tests/test_create_edit_office_hours_view.py

```diff
@@ -276,6 +376,18 @@ def _assert_update_office_hour(self, office_hour):
             pk=office_hour.id)
         self.assertEqual(updated_office_hour.topics, self.updated_topics)
 
+    def _assert_office_hour_not_updated(self, office_hour):
+        updated_office_hour = MentorProgramOfficeHour.objects.get(
```

Comment:
Hm. Just realized that there's a field called `updated_at` :)

No need to spend time on it, but the following would probably work just as well:

```
def _assert_office_hour_not_updated(self, office_hour):
    previously_updated_at = office_hour.updated_at
    office_hour.refresh_from_db()
    self.assertEqual(previously_updated_at, office_hour.updated_at)
```